.github/workflows/pull-request.yml

```
@@ -1,25 +1,139 @@
+  XCODE_VERSION: "16.3"
 
 jobs:
-  pull-request:
+  prepare:
     runs-on: macos-15
-
+    outputs:
+      platforms: ${{ steps.platforms.outputs.platforms }}
+      scheme: ${{ steps.scheme.outputs.scheme }}
     steps:
     - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)